charts/camunda-platform/values/values-latest.yaml

```diff
@@ -13,7 +13,7 @@ connectors:
   # https://hub.docker.com/r/camunda/connectors-bundle/tags
   image:
     repository: camunda/connectors-bundle
-    tag: 8.4.3
+    tag: 8.4.4
```

Comment:
The tag is changed here as well. Renovate already does this process: https://github.com/camunda/camunda-platform-helm/pull/1301/files#diff-1eef702167416a7ff2ba0912d6f8427a08211d7556551d2c315fb9c9a42cefba

Reply:
So would you advice to remove tag changes here and merge the renovate PR first and then this one? I included the tag upgrade because the changes in this PR are required to run 8.4.4 and also do not make sense for previous versions, so merging the version bump separately would result in an inconsistent state, where connectors do not start properly.

Reply:
Hi @chillleader it's ok we will close out the renovate PR, no issues here.